src/context/toastNotificationProvider.tsx

```diff
@@ -19,7 +19,7 @@ import React, {
   useState,
 } from "react";
 
-const HIDE_NOTIFICATION_DELAY = 5_000;
+export const HIDE_NOTIFICATION_DELAY = 5_000;
```

Comment:
I think this can be reverted as well at this point.

Reply:
aah yes, will revert that :+1: